ansible/roles/ufw/tasks/proxy_ufw.yml

```diff
@@ -0,0 +1,35 @@
+# Rules for monoliths
+- name: allow web traffic
+  ufw:
+    rule: allow
+    direction: in
+    port: "{{ item }}"
+    proto: tcp
+  with_items:
+    - 443  # https
+    - 80   # http
+  when: is_monolith
+
+- name: allow ssh traffic
```

Comment:
Ah. Do we need to add a second factor on monoliths then? Like google-auth or something?
(probably for discussion later, but just flagging now)